templates/openshift/edxapp/dc/nginx.yml.j2

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: nginx
```

Comment:
All version of an app are grouped on th same app on OpenShift ?
Why do not include the deployment_stamp also ?
The app must be `edxapp` no ?

Reply:
Yes! and that will be part of my next refactoring. IMO this is not part of the scope of this PR.

Reply:
Ok